Libs/Optimize/ParticleSystem/itkParticleSystem.txx

```diff
@@ -400,6 +400,8 @@ ParticleSystem<VDimension>
     auto neg_projected = -projected;
     newpos = this->GetDomain(domain)->UpdateParticlePosition(startingPos, neg_projected);
     this->SetPosition(newpos, k, domain, threadId);
+    const ParticleDomain *domain = this->GetDomain(domain);
```

Comment:
is this correct? There might be some name shadowing going on.

My IDE says this function is never used however.

Reply:
Oops!

Reply:
Yeah, that's right, there's the Advanced particles splitting now.